lib/msf/base/sessions/command_shell.rb

```diff
@@ -544,8 +546,9 @@ def cmd_irb(*args)
     if expressions.empty?
       print_status('Starting IRB shell...')
       print_status("You are in the \"self\" (session) object\n")
-
+      Msf::Ui::Console::HistoryManager.push_context(history_file: nil,name: 'IRB')
```

Comment:
All the uses of the `name:` kwarg should consistently be using the symbol data type. Also this is a good reason why `history_file:` could default to `nil` in the signature. That would make this:

```suggestion
      Msf::Ui::Console::HistoryManager.push_context(name: :irb)
```